### PR TITLE
Made people table

### PR DESCRIPTION
**Pull request by mira1zu**
- [DEMO LINK](https://mira1zu.github.io/react_people-table-basics/)

### CODE REVIEW

**Review by AndriiNychaiuk — APPROVED** [review]
👍

File: src/components/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  return (
+    <>
+      <header className="mb-5">
+        <nav className="nav nav-pills nav-fill p-3 bg-light border rounded-4">
+          <NavLink className="nav-link" to="/" end>Home</NavLink>
+          <NavLink className="nav-link" to="people" end>People</NavLink>
+        </nav>
+      </header>
+
+      <Outlet />
```
Comment (illiarerun) [inline src/components/Header.tsx:14]: 🔥

File: src/components/PeopleTable.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          <th scope="col">Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map((person, i) => (
+          <PersonRow
+            key={person.slug}
+            person={person}
+            index={i + 1}
```
Comment (illiarerun) [inline src/components/PeopleTable.tsx:31]: let's not use indeces for `key` value, you can use `slug` or smth else

File: src/components/PeoplePage.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      {isError
+        ? (
+          <h3
+            className="text-center"
+          >
+            An error occured while loading people data
+          </h3>
+        )
+        : (
+          <>
```
Comment (illiarerun) [inline src/components/PeoplePage.tsx:49]: do you really need a fragment here?
```suggestion
```

**Review by illiarerun — APPROVED** [review]

**Pull request closed without merging**